test/k8s-integration/config/fs-sc-premium-default-netwrok.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-filestore-premium
+provisioner: filestore.csi.storage.gke.io
+parameters:
+  tier: premium
+volumeBindingMode: WaitForFirstConsumer
```

Comment:
This should probably be immediate binding mode? That will speed up the test slightly. Plus we'll probably want to move to immediate binding mode for the default deployment anyway.

Reply:
Done